indico/web/client/js/react/components/Select.jsx

```diff
@@ -0,0 +1,100 @@
+// This file is part of Indico.
```

Comment:
I noticed this really weird behavior in Firefox where it randomly selects text when repeatedly opening and closing the dropdown... could be a Firefox bug though, so I would not spend *too* much time on it...

![weird-ff-selection](https://github.com/user-attachments/assets/273de644-91b1-48b7-801d-a25d259dc325)

Reply:
Actually it seems to happen when moving the mouse cursor just the slightest bit while clicking.

Reply:
Well, that's a common issue with any custom widget. We can mask it with CSS so at least it's not visible, but not sure if selection can be outright prevented or if it's even worth it.

Reply:
I never noticed it with other widgets... Would deferring the closing of the dialog help, so the click event has already been fully handled by the time it closes?

Reply:
> I never noticed it with other widgets...

Happens quite often actually.

> Would deferring the closing of the dialog help, so the click event has already been fully handled by the time it closes?

Will see what I can do.